main/reference/zoe-api/zoe-data-types.md

```diff
@@ -44,11 +46,15 @@ const myAmountKeywordRecord =
 ```
 ## Handle
 
-**Handles&** within Zoe have a slightly different definition than in regular JavaScript. They are **Far** objects without any methods whose only useful property are their unique unforgeable identities. They're often created in order to designate some other object, where the **Handles** can be passed around as reliable designators without giving access to the designated ojects.
+**Handles** within Zoe have a slightly different definition than in regular JavaScript. They are **Far**
```

Comment:
Javascript has *handlers*, but I don't think it has *handles*. Would it make more sense to say something more like "Handles within Zoe are unrelated to Javascript event handlers"?

Reply:
Oh... handles are so ubiquitous in other languages, I just assumed that JavaScript had them too. Since JavaScript doesn't have them, that first sentence is pretty much unnecessary, and I deleted it. 

I don't see any reason to mention JavaScript's event handlers at all; that can create a link in readers' minds even if we use your suggested passage.